### DISCUSSION TITLE
How to plan operations without intermediate pauses

### DISCUSSION THREAD

**Discussion by matteocarcano** [frePPLe/frepple#647] [Q&A] [answered]
Hi, 
i would like to ask you how can I set model to plan operations only if they can start and end in the same daily bucket.
e.g.: daily working hours: 6.00 am - 10.00 pm.
There's one operation that starts at 8.00 am and finishes at 4.00 pm, so there are only 6 hours free available. How can i set Frepple as it plans an operation only if its duration is less than 6 hours? This because started operation can't stay idle during the night pause.

Thank you in advance.

bye

**Comment by jdetaeye** [comment, marked as answer]
You can use a "maximum calendar" for your resources, and make sure that the capacity is 0 during off-shift periods.

There is a subtle difference between the "available calendar" and "maximum calendar":   A manufacturing order can be planned across an unavailable period and it's duration will be "strethed" for the unavailable time (ie you start today and finish it tomorrow).   But a manufacturing order can NOT be planned across a period where the resource max size is 0.

There is also a danger with a "maximum calendar".   If you set the max size to 0 each day during the night, we will not be able to plan an operation that takes eg 48 hours.  We'll never find a period of 48 hours where the resource size stays above 0, and your demands will get unplanned.

**Reply by matteocarcano** [reply]
Thank you very much for the quickly reply, now it's clear.

Have a nice day
